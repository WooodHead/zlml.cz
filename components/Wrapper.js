// @flow

type Props = {
  children?: Object,
};

export default ({ children }: Props) =>
  <div className="wrapper">
    {children}
    <style jsx>{`
      .wrapper {
        background: white;
        max-width: 1100px;
        margin: 0 auto;
        padding-right: 12px;
        padding-left: 12px;
        box-shadow: 0 1px 2px #aaa;
        margin-top: 12px;
      }

      @media screen and (min-width: 720px) {
        .wrapper {
          padding-right: 24px;
          padding-left: 24px;
        }
      }
    `}</style>
  </div>;
