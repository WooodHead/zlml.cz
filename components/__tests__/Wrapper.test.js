// @flow

import React from 'react';
import renderer from 'react-test-renderer';

import Wrapper from '../Layout';

it('renders correctly', () => {
  expect(
    renderer.create(<Wrapper>WRAPPED</Wrapper>).toJSON(),
  ).toMatchSnapshot();
});
