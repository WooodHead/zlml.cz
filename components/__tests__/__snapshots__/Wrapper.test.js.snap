// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="jsx-2213994124 wrapper"
  >
    WRAPPED
    <footer>
      <p
        style={
          Object {
            "fontSize": "2rem",
          }
        }
      >
        By the way - this website is also
         
        <a
          href="https://github.com/mrtnzlml/zlml.cz"
        >
          open-sourced
        </a>
        .
      </p>
    </footer>
  </div>
</div>
`;
